app/Concerns/HasBans.php

```diff
@@ -0,0 +1,152 @@
+<?php
```

Comment:
A mix of Strict return types and Docblock return types - might be good to be consistent either way?